### PR TITLE
kubeadm reset command titles should not be translated

### PR DESCRIPTION
**Pull request by xieyanker**
supplement "kubeadm reset" translation

### CODE REVIEW

**Comment by xichengliudui** [comment]
these are part of the command and do not need to be translated.
@xieyanker thank you for your contributiont

**Comment by xieyanker** [comment]
@xichengliudui @pigletfly Hi, can this PR be approved to merge? It seems no `lgtm` and `approved` labels.

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-reset-phase.md [hunk 1]
```diff
@@ -54,9 +54,9 @@ Using this phase you can remove this control-plane node from the ClusterStatus o
 {{< /tabs >}}
 
 <!--
-## kubeadm reset 移除 etcd 成员的阶段
+## kubeadm reset phase remove-etcd-member
 -->
-## kubeadm reset phase remove-etcd-member {#cmd-reset-phase-remove-etcd-member}
+## kubeadm reset 移除 etcd 成员的阶段 {#cmd-reset-phase-remove-etcd-member}
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-reset-phase.md]: this line should not be translated ...

Comment (tengqm) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-reset-phase.md]: Line 73 translation is incorrect. We are not supposed to translate the command itself.

Comment (xieyanker) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-reset-phase.md]: Thanks for your reply! In fact, the changes I made were to be consistent with other heading styles.  As you say, this means that some other command titles should not be translated, these includ `kubeadm init phase`,`kubeadm join phase`,`kubeadm reset phase` and `kubeadm upgradephase`. If you are sure to do this, I can fix all of them.

Comment (tengqm) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-reset-phase.md]: Yes. Please reread the article. The original translation was wrong... they should not be translated.

Comment (xieyanker) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-reset-phase.md]: @tengqm I have repushed, and fix all of wrong "kubeadm reset command titles".

**Comment by tengqm** [comment]
thanks
/lgtm

**Comment by xieyanker** [comment]
@xichengliudui @pigletfly Hi, can this PR be approved to merge?

**Pull request merged**